packages/common/src/ui/components/NavLink.tsx

```diff
@@ -48,9 +62,9 @@ export const AppNavLink: FC<ListItemLinkProps> = props => {
 
   return (
     <li>
-      <Tooltip title={text || ''}>
+      <Tooltip disableHoverListener={drawer.isOpen} title={text || ''}>
```

Comment:
that's probably fair enough. no need to show the tooltip on the label which has the same text!